src/frontendHighlighterApp/sass/app.scss

```
@@ -51,6 +51,9 @@ body {
 		outline: dashed 4px transparent !important;
 		outline-offset: 5px !important;
 		outline-color: magenta !important;
+		// White ring between the element and the magenta outline so the indicator
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)